parser/parser.go

```diff
@@ -123,6 +131,29 @@ func ParseFile(r io.Reader) (*File, error) {
 				role = b.String()
 			case stateComment, stateNil:
 				// pass
+			case stateVersion:
+				s, ok := unquote(strings.TrimSpace(b.String()))
+				if !ok {
+					if _, err := b.WriteRune(r); err != nil {
+						return nil, err
+					}
+
+					continue
+				} else if isSpace(r) {
+					return nil, errInvalidVersion
+				}
+
+				if s[0] != 'v' {
+					s = "v" + s
+				}
+
+				if !semver.IsValid(s) {
+					return nil, errInvalidVersion
+				}
+
+				cmd.Args = semver.Canonical(s)
+				f.Commands = append(f.Commands, cmd)
```

Comment:
```suggestion
				f.Commands = append(f.Commands, semver.Canonical(s))
```

Reply:
This replaces the cmd object with just the version, which isn't what we want right

Reply:
oops yeah my bad